types/dockerode/index.d.ts

```diff
@@ -567,9 +567,11 @@ declare namespace Dockerode {
             };
         };
         Mounts: Array<{
+            Type: string;
```

Comment:
Should this be

```suggestion
            Type: "bind" | "volume" | "image" | "tmpfs" | "npipe" | "cluster";
```

?

Reply:
Looks even better but I haven't checked all possible values

Reply:
I pulled that list from the swagger.yaml you linked!

Reply:
Ok, I saw some mistakes in this swagger, I opened 2 issues about it this week so I'm a bit cautious about using it as is.